### ISSUE TITLE
Docs Site Watermarks on (beta, limited release, etc) should be more visible

### ISSUE THREAD

**Issue by bradleycamacho** [newrelic/docs-website#1599]
We’ve received some feedback that the watermarks (beta, limited release, etc) are a bit too faint and hard to see. It's so hard that someone in dark mode didn't even notice a doc had a "beta" watermark. 

We should try to find a way to make these easier to see, whether this is increasing the size, contrast, or font. 

### Acceptance Criteria

- [ ] Create a new component for these docs pages that would be a banner element that would be sticky and clearly articulate the the state of the doc for BETA docs.
- [ ] the element should be added to the Docs Repo only, and not be part of the global theme as it's only applicable to the Docs Site.

### Other notes

We Improve on this with a `stop gap` improvement via: https://github.com/newrelic/docs-website/pull/1600

### Example images below:

<img width="1032" alt="Screen Shot 2021-03-30 at 11 37 35 AM" src="https://user-images.githubusercontent.com/42678939/113041209-8df43300-914e-11eb-854d-05add595f47a.png">
<img width="1642" alt="Screen Shot 2021-03-30 at 11 39 30 AM" src="https://user-images.githubusercontent.com/42678939/113041214-8fbdf680-914e-11eb-86f3-8dd331dfd7b4.png">

**Comment by bradleycamacho** [comment]
Note that #1600 does not close this, but it simply provides a stop-gap solution. We should look into making the watermark more accessible in general by changing the colors, contrast, etc.

**Comment by aswanson-nr** [comment]
@jpvajda and @bradleycamacho I took a look at this as well. The contrast is completely controlled by the color and how it looks against the background color of the theme. The light theme color is `#eeeeee` and the dark theme color, controlled by `--color-dark-100` is `#345360`. I'm hesitant to make any changes without discussing new color choices with a designer.

**Comment by aswanson-nr** [comment]
Reopened since the #1600 was just a stop gap.

**Comment by austin-schaefer** [comment]
Duplicated in #1684

**Comment by aswanson-nr** [comment]
Totally missed this issue! adding my content here.

@bradleycamacho Suggested this change to the watermark.

```javascript
const Watermark = ({ text }) => (
  <div
    data-swiftype-index={false}
    css={css`
      position: fixed;
      left: 25%;
      top: 75%;
      transform: translate(-25%, -75%);
      color: #eeeeee;
      font-size: 18rem;
      transform: rotate(-25deg);
      z-index: -1;
      top: calc(var(--global-header-height) + 8rem);
      user-select: none;
      white-space: pre;
      line-height: 1;
      .dark-mode & {
        color: var(--color-dark-100);
      }
    `}
  >
    {text}
  </div>
);
```
<img width="1511" alt="Screen Shot 2021-04-05 at 10 24 03 AM" src="https://user-images.githubusercontent.com/70179303/113605325-575d6300-95fb-11eb-9e7f-ad90147439fd.png">
<img width="1511" alt="Screen Shot 2021-04-05 at 10 24 12 AM" src="https://user-images.githubusercontent.com/70179303/113605329-59272680-95fb-11eb-8b70-8f3c0c4dbf17.png">
<img width="1510" alt="Screen Shot 2021-04-05 at 10 26 29 AM" src="https://user-images.githubusercontent.com/70179303/113605333-5a585380-95fb-11eb-87c4-c1b6bf5bf42d.png">
<img width="1511" alt="Screen Shot 2021-04-05 at 10 26 39 AM" src="https://user-images.githubusercontent.com/70179303/113605339-5af0ea00-95fb-11eb-82db-a4d05a54189b.png">

**Comment by jpvajda** [comment]
As a solution to this problem I was considering making the size of the font much smaller and repeating the text as a background of a doc. so the words would just span across the page as a "background image" if you will. @aswanson-nr  thoughts?

**Comment by jpvajda** [comment]
@bradleycamacho  also thoughts.. 😄

**Comment by aswanson-nr** [comment]
@jpvajda I like the idea! I could see some performance issues with rendering the same thing repeatedly across the page, but that might just be over optimization on my part.

A banner that sticks to the nav bar on top/bottom could also work. Although I'm not familiar with that code.

**Comment by jpvajda** [comment]
@aswanson-nr I was thinking a simple `JS for loop` could just place the element on the page as I was doing some research on how to apply text as background image. as for a page banner, we could also take that approach. I feel I could get the work done to apply the text as repeating background, but the a new component probably is bit beyond the time I have available to sling code. 😄 

```
// need to figure out how to make this loop work for the watermark
for (var i = 0; i < 50; i++) {}

const Watermark = ({ text }) => (
  <div
    data-swiftype-index={false}
    css={css`
      position: fixed;
      left: 25%;
      top: 75%;
      transform: translate(-25%, -75%);
      color: #eeeeee;
      font-size: 18rem;
      transform: rotate(-25deg);
      z-index: -1;
      top: calc(var(--global-header-height) + 8rem);
      user-select: none;
      white-space: pre;
      line-height: 1;
      .dark-mode & {
        color: var(--color-dark-100);
      }
    `}
  >
    {text}
  </div>
);
```

**Comment by jpvajda** [comment]
updated the issue with the approach the team discussed.

**Comment by austin-schaefer** [comment]
@rhetoric101 : Can you take a look at this one? I think we had a fix IDed for this, right?

**Comment by urbiz-nr** [comment]
Closed #1599 via #3834 and #3835. See DOC-7243 for more details.
